package-lock.json

```diff
@@ -40,7 +40,7 @@
     },
     "node_modules/@ampproject/remapping": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz",
+      "resolved": "https://na.artifactory.swg-devops.com/artifactory/api/npm/wcp-alchemy-containers-team-npm-virtual/@ampproject/remapping/-/remapping-2.2.0.tgz?dl=https%3A%2F%2Fregistry.npmjs.org%2F%40ampproject%2Fremapping%2F-%2Fremapping-2.2.0.tgz",
```

Comment:
Accidental switch to use `artifactory` -- check your npm config, revert package-lock.json, and re-do the `npm update`